tests/functional/workloads/cnv/test_vm_snapshot_cloning_ops.py

```diff
@@ -49,11 +83,32 @@ def test_vm_clone(
         )
         vm_list = vm_objs_def + vm_objs_aggr
         log.info(f"Total VMs to process: {len(vm_list)}")
-        for index, vm_obj in enumerate(vm_list):
+        for vm_obj in vm_list:
             log.info(
                 f"Starting I/O operation on VM {vm_obj.name} using "
                 f"{file_paths[0]}..."
             )
+            # Expand PVC if `pvc_expand_before_snapshot` is True
+            pvc_obj = vm_obj.get_vm_pvc_obj()
+            new_size = 50
+            if pvc_expand_before_clone:
+                try:
+                    pvc_obj.resize_pvc(new_size=new_size, verify=True)
+                    time.sleep(30)
```

Comment:
what is the need for the sleep here? can we avoid this hardcoded  sleep?

Reply:
Done